server/src/main/resources/rules/rules.js

```diff
@@ -2595,9 +2600,11 @@ var Autobind = {
              * Returns the role that the stack will cover, which the consumer requires.
              */
             get_common_role: function(role) {
-                if (role == null) {
+                if (!role || !role.trim()) {
```

Comment:
Just noticed this change. I don't think the `!role.trim()` is needed here; it will always return `false`. `if (!role) {` should be enough. Same thing with the addon if statement above. After this we can merge!

Reply:
This was intended because if the role has a white-space and on trim it finds empty.

Reply:
Ah , I see. That makes sense then